src/librustc_mir/interpret/snapshot.rs

```diff
@@ -305,7 +305,7 @@ impl_stable_hash_for!(enum crate::interpret::eval_context::StackPopCleanup {
 #[derive(Eq, PartialEq)]
 struct FrameSnapshot<'a, 'tcx> {
     instance: &'a ty::Instance<'tcx>,
```

Comment:
This also seems unnecessary, an instance should be cheap to copy around.

Reply:
File a cleanup issue please so the comment doesn't get lost. :)